torchci/lib/bot/retryBot.ts

```diff
@@ -54,27 +53,42 @@ function retryBot(app: Probot): void {
       // or a widespread outage that wouldn't be helped by retries
       return;
     }
+
+    // @ts-expect-error - we don't have types for these
+    let jobsDoesntFailStepsLike = (job, predicate: (step: any) => boolean) => {
```

Comment:
Variable name sounds confusing. It's common practice to start  boolean predicate names with `is`/`does` (so it looks like a question: `isEven`, `didFail`, etc..
```suggestion
    let doesLookLikeInfraFailure = (job, predicate: (step: any) => boolean) => {
```

Reply:
Ooo, I like this name!